### PR TITLE
Django 1.8 support

### PR DESCRIPTION
**Pull request by bogdal**
This PR makes `saleor` compatible with `Django 1.8`.

### CODE REVIEW

**Comment by mociepka** [comment]
Tests failed :/

**Comment by bogdal** [comment]
Yes, I know. That's why this PR has `in progress` label. I'm waiting for the new release of `django-emailit` package.

**Comment by mociepka** [comment]
Stefan merged it a day before yours comment.

**Comment by bogdal** [comment]
Today Stefan released new version on pypi https://pypi.python.org/pypi/django-emailit/0.2.2

File: saleor/core/utils/__init__.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def build_absolute_uri(location, is_secure=False):
     except AttributeError:
         raise ImproperlyConfigured('You need to specify CANONICAL_HOSTNAME in '
                                    'your Django settings file')
-    if not absolute_http_url_re.match(location):
+    if not re.compile(r"^https?://", re.I).match(location):
```
Comment (patrys) [inline saleor/core/utils/__init__.py]: What happened here?

Comment (bogdal) [inline saleor/core/utils/__init__.py]: `absolute_http_url_re` is no longer available in the latest version of Django

Comment (patrys) [inline saleor/core/utils/__init__.py]: But what is the point of compiling the regex at this point? It's just as slow as `re.match(…)` as you recompile each time the condition is checked.

**Pull request merged**